src/views/List.jsx

```diff
@@ -7,11 +8,11 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{data.map((item) => (
+					<Fragment key={item.id}>
+						<ListItem name={item.name} />
+					</Fragment>
+				))}
```

Comment:
Good job team! I can also see you used the `key` to help react identify each item, that's a good coding practice!💥